modules/ROOT/pages/metrics.adoc

```diff
@@ -0,0 +1,56 @@
+// start {partialsdir}/attributes.adoc[]
+include::_partials/attributes.adoc[]
+// end {partialsdir}/attributes.adoc[]
+
+:toclevels: 4
+
+:toc:
+
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+:context: {metrics-service}
+ifdef::context[:parent-context: {context}]
+
+:installation-link: getting-started.html
+:docs-link: https://www.keycloak.org/documentation.html
+
+
+[id='a-collection-of-modules']
+
+= Using the {metrics-service} Mobile Service
+
+[[introduction]]
+The {metrics-service} service allows you to gather metrics on mobile apps, device versions, device security checks and back-end mobile service usage.
```

Comment:
text under toc without heading, do we generally want this @finp ?

Reply:
@pmacko1  It makes sense ATM in order to get stuff downstream, worth revisiting after tech preview